src/cf-forms/src/atoms/label.less

```diff
@@ -14,5 +14,9 @@
 
         // Overwrites heading-4 margin.
         margin-bottom: unit( 10px / @font-size, em );
+
+        &_helper {
```

Comment:
This will need it's full class name otherwise it'll resolve to `a-label__heading_helper` rather than `a-label__heading a-label_helper` that you want

Reply:
Oh shoot! Dumb mistake. Fixed!